Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonHelpCommand.cs

```diff
@@ -49,6 +49,50 @@ public class KryptonHelpCommand : KryptonCommand
          [AllowNull, Category(@"Appearance"), Description(@"State specific images for the help button."), DesignerSerializationVisibility(DesignerSerializationVisibility.Hidden)]
          public ButtonImageStates? ImageStates { get => _imageStates ?? new(); set { _imageStates = value; UpdateImageStates(KryptonManager.InternalGlobalPaletteMode); } }*/
 
+        [AllowNull]
```

Comment:
These do not need `[AllonwNull]` because they can return null as well.
Also why are the `set`'s not expressions

Reply:
Addressed feedback and `set`'s are now `private` as images _should_ be theme based